charts/victoria-logs-collector/templates/configmap.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "victoria-logs-collector.fullname" . }}-config
+  labels:
+    app.kubernetes.io/name: {{ include "victoria-logs-collector.fullname" . }}
+data:
+  vector.yaml: |
+    data_dir: /vl-collector
+
+    sources:
+      k8s:
+        type: kubernetes_logs
+        pod_annotation_fields:
+          # Disable "pod_ips" as "pod_ip" already includes required info
+          pod_ips: ""
+
+    transforms:
+      json_parser:
+        type: remap
+        inputs: [k8s]
+        source: |
+          .message = parse_json(.message) ?? .message
+
+    {{- if empty .Values.remoteWrite }}
+      {{ fail "specify at least one remoteWrite with valid url"}}
+    {{ end }}
+
+    sinks:
+    {{- range $i, $rw := .Values.remoteWrite }}
+      {{/* Validate remoteWrite object */}}
+      {{- if empty $rw.url }}
+        {{ fail (printf "'url' field must be set for remoteWrite[%d]" $i) }}
+      {{- end }}
+      {{- range $rwKey, $_ := $rw }}
+        {{- if and (ne $rwKey "url") (ne $rwKey "basicAuth") (ne $rwKey "accountID") (ne $rwKey "projectID") (ne $rwKey "tls") }}
+          {{ fail (printf "unknown field %q for remoteWrite[%d]" $rwKey $i) }}
+        {{- end }}
+      {{- end }}
+      remote_write_{{ $i }}:
+        type: http
+        inputs: [json_parser]
+        uri: {{ printf "%s/insert/jsonline" (trimSuffix "/" $rw.url) | quote }}
+        encoding:
+          codec: json
+        framing:
+          method: newline_delimited
+        healthcheck:
+          enabled: false
+        request:
+          headers:
```

Comment:
I am not sure we need to expose the `headers` section, because it complicates setting default headers like `VL-Stream-Fields` and so on. What do you think?

Reply:
I think, current configuration is fine and there is no need to expose headers  at .Values.